### DISCUSSION TITLE
Replay analysis HD issue

### DISCUSSION THREAD

**Discussion by YoYo745** [ppy/osu#29795] [Ideas]
When using replay analysis on replay with Hidden mod you cannot see the hitcircle, thus making the analysis of cursor placement impossible. For this it would be nice to have option to disable Hidden mod. Adding a video with example.

https://github.com/user-attachments/assets/ab1e95d4-70b7-496d-b1bd-206fcf9a3313

**Comment by peppy** [comment]
Probably this thing: https://github.com/ppy/osu/pull/19925
